jwst/resample/resample_spec.py

```diff
@@ -99,6 +99,9 @@ def build_nirspec_output_wcs(self, refmodel=None):
         else:
             refmodel = self.input_models[0]
 
+        refmodel_data = refmodel.data.copy()
+        refmodel_data[refmodel_data < 0.] = np.NaN
```

Comment:
One possible way to improve the logic would be to find the minimum value in data and subtract it from data:
```suggestion
        min_refmodel_data = np.nanmin(refmodel_data)
        refmodel_data -= min_refmodel_data
```

Reply:
Another approach would be to add back subtracted sky if that value was recorded in the metadata.

Reply:
Unfortunately this is not a case of having a constant background subtracted. It's images with the spectra in different places that've been subtracted from one another, so you're left with both positive and negative traces of what is essentially the same spectrum.

Reply:
I implemented your suggested change to renormalize to the min value. Seems to work well without explicitly excluding some of the data (like my initial approach did).